.github/workflows/ci.yml

```diff
@@ -23,6 +23,7 @@ env:
   # Override the default rust compilation profile made in `build.py`,
   # We prefer using one that is more fit to the CI.
   CARGO_PROFILE: integration
+  VCPKG_ROOT: $VCPKG_INSTALLATION_ROOT
```

Comment:
:thinking: 
What this env variable do ? 
why it seems like it's defined to another env variable `$VCPKG_INSTALLATION_ROOT` ?

Reply:
openssl-sys complained during previous runs that it could not find a vcpkg installation, and that we could provide a path using the environment variable `VCPKG_ROOT`.

The image used by our actions provides a similar variable but with a different name (`VCPKG_INSTALLATION_ROOT`). 

I did this instead of specifying a path for openssl, as this is subject to change

Reply:
Note: it is possible to hardcode this to `C:\vcpkg` if defining a variable to another existing variable does not work

Reply:
in github action variable / env expansion use the following syntax

```yaml
${{ env.VCPKG_INSTALLATION_ROOT }}
```

Reply:
Can you add a comment to indicate that this env var is for windows only and for openssl ?

Reply:
Thank you for the correct syntax.

Fixed in [c37fbec](https://github.com/Scille/parsec-cloud/pull/3629/commits/c37fbec248258ff2f5ccd55eb8c16c2a14afc6c0)